features/com.google.cloud.tools.eclipse.suite.e45.feature/feature.xml

```diff
@@ -3,18 +3,19 @@
       id="com.google.cloud.tools.eclipse.suite.e45.feature"
       label="Google Cloud Platform for Eclipse 4.5"
       version="0.1.0.qualifier"
-      provider-name="Google, Inc.">
+      provider-name="Google, Inc."
+      license-feature="com.google.licenses.apache_v2">
 
    <description>
-      Support development of Google Cloud Platform-based Projects in Eclipse 4.5
+      %description
    </description>
 
    <copyright>
-      Copyright 2016
+      %copyright.text
```

Comment:
I only see description set in .properties. Where are copyright.text, license, and license.text?